tests/integration/defs/perf/disagg/report.py

```
@@ -0,0 +1,274 @@
+            print(f"Found match: {match.group(0)[:100]}...")
+            print(f"All groups: {match.groups()}")
+            print(f"Number of groups: {len(match.groups())}")
+
+            if len(match.groups()) < 3:
+                print(f"Warning: Expected 3 groups but got {len(match.groups())}")
+                continue
+
+            try:
+                values = [float(x) for x in match.groups()[:-1]]
+                concurrency = int(match.groups()[-1])  # Use groups()[-1] instead of group(-1)
+                item = dict(zip(metric_names, values))
```

Ruff (0.14.4)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)